PocketKit/Sources/PocketKit/Main/RegularMainCoordinator.swift

```diff
@@ -88,163 +75,75 @@ class RegularMainCoordinator: NSObject {
     }
 
     func showInitialView() {
-        model.$isCollapsed.receive(on: DispatchQueue.main).sink { [weak self] isCollapsed in
+        sizeClassObserver = model.$isCollapsed.sink { [weak self] isCollapsed in
```

Comment:
curious to see why this way is preferred

Reply:
Well, we had a property called `longSubscriptions` which was of type `[AnyCancellable]`. I realized that we only had one "long subscription" so we didn't need to have an array, and I felt the name `sizeClassObserver` was more meaningful considering what the property was for.